pkgs/tools/misc/idevicerestore/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig
+, curl
+, libimobiledevice
+, libirecovery
+, libzip
+, libusbmuxd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "idevicerestore";
+  version = "2019-02-14";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "libimobiledevice";
+    repo = pname;
+    rev = "8a882038b2b1e022fbd19eaf8bea51006a373c06";
+    sha256 = "17lisl7ll43ixl1zqwchn7jljrdyl2p9q99w30i6qaci71mas37m";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkgconfig
+
+    curl
+    libimobiledevice
+    libirecovery
+    libzip
+    libusbmuxd
+    # Inhereting other dependencies from `libimobiledevice`
```

Comment:
Not really related: libimobiledevice has a [.pc](https://github.com/libimobiledevice/libimobiledevice/blob/92c5462adef87b1e577b8557b6b9c64d5a089544/src/libimobiledevice-1.0.pc.in) but it's not in the output

Reply:
@worldofpeace I don't quite understand your comment; I put my comment there because other deps specified in https://github.com/libimobiledevice/idevicerestore/blob/8a882038b2b1e022fbd19eaf8bea51006a373c06/README#L20 were somehow not needed to build.

Reply:
Random comment was labelled accordingly :smile:

Reply:
I've also improved the comment.

> Not really related: libimobiledevice has a [.pc](https://github.com/libimobiledevice/libimobiledevice/blob/92c5462adef87b1e577b8557b6b9c64d5a089544/src/libimobiledevice-1.0.pc.in) but it's not in the output

At least on latest `master`, it has a `.pc` file in the output`.